lib/Service/ActivityService.php

```diff
@@ -212,14 +189,14 @@ public function getItem(string $id): ACore {
 		}
 
 		$requests = [
-			$this->notesRequest
+			'Note'
```

Comment:
Any reason this is an array that is iterated over and not just one call? Also lets use Note::TYPE instead of the hardcoded string.